Topics/Tech_Stacks.md

```diff
@@ -1,3 +1,52 @@
 ## Tech Stacks
 
 ### [Learning MySQL for databases](./Tech_Stacks/Learning_MySQL.md)
+
+## Tech Stacks
+
+### CSS (Cascading Style Sheets)
+
+CSS, or cascding style sheets, are used to add styles, like spacing and layout, to HTML web pages. While CSS can be done through in-line styles, it is common convention to have a style sheet for the webpage, or the style tag. 
+
+CSS rules consist of a selector and a declaration block, where the selector points to the HTML element or set of elements you want to style and the declaration denotes the style. 
+
+<img width="444" alt="Capture" src="https://user-images.githubusercontent.com/81656509/226232503-8d496905-038e-4f61-b17f-d4f702f75ecf.PNG">
+
+
+Image source: https://www.w3schools.com/css/css_syntax.asp 
```

Comment:
```suggestion
[Image source](https://www.w3schools.com/css/css_syntax.asp)
```

Can you use use Markdown links when referring to a link? It's easier to read, in my opinion.

Reply:
Thank you for your suggestions. I added all the changes you requested in the commits below.